과제3/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>디자인과제3</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="box">
```

Comment:
모든 컨텐츠를 감싸고 있는 div 이므로 클래스명을 box보다는 wrapper 또는 container로 하는 것이 좋습니다.
```suggestion
    <div class="wrapper">
```

Reply:
어떤 content 를 감싸는 div 박스인지 명시하면 좋습니다. (ex. main-box, main-wrapper)